dev-utils/karma.js

```diff
@@ -39,7 +39,7 @@ const { getWebpackConfig, BUNDLE_TYPES } = require('./build')
 const polyfills = 'test/polyfills.+(js|ts)'
 
 const specPattern =
-  'test/{*.spec.+(js|ts),!(e2e|integration|node|bundle|types)/*.spec.+(js|ts)}'
+  'test/{*.spec.+(js|ts),!(e2e|integration|node|bundle|types)/**/*.spec.+(js|ts)}'
```

Comment:
The regex was not taking into consideration tests within subfolders, such the one added in https://github.com/elastic/apm-agent-rum-js/pull/1146/files#diff-61e07f7c1eabb9de7994cdf0f9a51bf3ced2db35ec5ceabf43966b04427f84c5